libs/server-sdk/src/client_impl.cpp

```diff
@@ -111,10 +111,11 @@ ClientImpl::ClientImpl(Config config, std::string const& version)
       work_(boost::asio::make_work_guard(ioc_)),
       memory_store_(),
       status_manager_(),
```

Comment:
This is saying: store updates in the `memory_store`, and also read flag data from the `memory_store`.